controllers/fio/job.go

```diff
@@ -33,14 +33,14 @@ func NewJob(cr *perfv1alpha1.Fio) *batchv1.Job {
 		Namespace: cr.Namespace,
 	}
 
-	fioCmdLineArgs := []string{}
-	fioCmdLineArgs = append(fioCmdLineArgs,
-		qsplit.ToStrings([]byte(cr.Spec.CmdLineArgs))...)
-	fioCmdLineArgs = append(fioCmdLineArgs, cr.Spec.BuiltinJobFiles...)
+	fioArgs := []string{}
```

Comment:
If we are touching this I'd recommend dropping the fio prefix as it brings no value (there is no otherappArgs in this context). This is true for all the other benchmarks.

Reply:
Modified all stones to use `args` as variable name.